src/custom/state/orders/mocks.ts

```diff
@@ -0,0 +1,180 @@
+import { useEffect, useRef } from 'react'
+import { batch } from 'react-redux'
+
+import { Order, OrderStatus, OrderKind } from './actions'
+import { useActiveWeb3React } from 'hooks'
+import { useAddPendingOrder, usePendingOrders, useFulfillOrder } from './hooks'
+import { useSelectedTokenList } from 'state/lists/hooks'
+import { useAddPopup } from '@src/state/application/hooks'
+import { TokenInfo } from '@uniswap/token-lists'
+
+const randomNumberInRange = (min: number, max: number) => {
+  return Math.random() * (max - min) + min
+}
+
+// Infinity to not trigger condition
+const randomIntInRangeExcept = (min: number, max: number, exception = Infinity) => {
+  let num = Math.floor(randomNumberInRange(min, max))
+  // >= because we Math.floor
+  if (num >= exception) ++num
+  return num
+}
+
+const getRandomElementFromArray = <T>(array: T[]): T => {
+  const ind = randomIntInRangeExcept(0, array.length - 1)
+  return array[ind]
+}
+
+const getTwoRandomElementsFromArray = <T>(array: T[]): [T, T] => {
+  const ind1 = randomIntInRangeExcept(0, array.length - 1)
+  const ind2 = randomIntInRangeExcept(0, array.length - 1, ind1)
+
+  return [array[ind1], array[ind2]]
+}
```

Comment:
The utils above could be moved to sth like `util/random.ts`

Reply:
But we won't need them at all when we no longer need the mock
Delete them all together then

Reply:
maybe :) 
was a nit, my thinking is there might be other mocks. Also is pretty general. whatever. 

Also, I'm not so sure we will delete this, it seems pretty useful, even with the contract there